src/containers/CreateStreamMainCard/index.tsx

```diff
@@ -39,14 +39,14 @@ const CreateStream = () => {
   const [isConfirm, setIsConfirm] = useState(false);
 
   const { address } = useAppSelector((state) => state.user);
-  const usrInfo = useAppSelector((state) => state.user?.info?.balances[0]);
+  const userInfo = useAppSelector((state) => state.user?.info?.balances[0]);
```

Comment:
`userInfo` is not a good name for this. Rename it both here and in validate form and the `UserInfo` type.
Instead, you can rename it to HorizonAsset type and `xlmAsset`